app/index.html

```diff
@@ -24,6 +24,7 @@
             <div class="header-right" id="actions">
               <span class="link" ng-click="app.refresh($event)" title="Refresh Destiny Data"><i ng-class="{'fa-spin': loadingTracker.active()}" class="fa fa-refresh"></i></span>
               <span class="link" ng-disabled="loadingTracker.active()" ui-sref="best" title="MinMax"><i class="fa fa-bar-chart"></i></span>
+              <span class="link" ng-click="app.downloadCsvFiles()" ng-disabled="loadingTracker.active()" title="Download Csv"><i class="fa fa-table"></i></span>
```

Comment:
I'd prefer if we hid this in "settings" or "about" or something. IMO it doesn't deserve a header button.